packages/effect/src/Either.ts

```diff
@@ -710,3 +710,94 @@ export const gen: Gen.Gen<EitherTypeLambda, Gen.Adapter<EitherTypeLambda>> = (f)
     return right(state.value)
   }
 }
+
+// -------------------------------------------------------------------------------------
+// do notation
+// -------------------------------------------------------------------------------------
+
+/**
+ * @since 2.4.0
+ * @category models
+ */
+export type MergeRecord<K, H> = {
```

Comment:
I noticed that there's already a `MergeRecord` in `Effect.ts`. I think we can relocate this helper to `Types.ts` and import it from there

Reply:
That certainly sounds like a good idea 👍

Reply:
Done ✅